build-config/arch/armv7a.make

```diff
@@ -13,6 +14,8 @@ CROSSPREFIX ?= $(TARGET)-
 CROSSBIN    ?= $(XTOOLS_INSTALL_DIR)/$(TARGET)/bin
 
 KERNEL_ARCH		= arm
+KERNEL_LOAD_ADDRESS	?= 0x61008000
+KERNEL_ENTRY_POINT	?= 0x61008000
```

Comment:
We are almost there.  I would make the default here be 0x6000_8000.  That is value we want people to use by default.

The handful of legacy platforms (3 machines I think) would need to set these to 0x6100_8000 in their machine.make:
```
accton/accton_as4610_30/machine.make:ONIE_ARCH ?= armv7a
accton/accton_as4610_54/machine.make:ONIE_ARCH ?= armv7a
dni/dni_3048up/machine.make:ONIE_ARCH ?= armv7a
```

Then all new machines going forward do not have to do anything special.

The way it is written here, hardware vendors have to go out of their way to "do the right thing".  I want to the "right thing" to be the default.